fs/littlefs/lfs_vfs.c

```diff
@@ -1531,7 +1531,7 @@ static int littlefs_mkdir(FAR struct inode *mountpt, FAR const char *relpath,
 
   if (len > 0 && relpath[len - 1] == '/')
     {
-      path = lib_get_pathbuffer();
+      path = lib_get_tempbuffer(PATH_MAX);
```

Comment:
i feel it's simpler to make
```
#define lib_get_pathbuffer() lib_get_tempbuffer(PATH_MAX)
```
instead of modifying all callers.

Reply:
> i feel it's simpler to make
> 
> ```
> #define lib_get_pathbuffer() lib_get_tempbuffer(PATH_MAX)
> ```
> 
> instead of modifying all callers.

that way we can go back to the fixed-sized allocataion easily if/when this change turned out to be a bad idea. :-)